packages/twenty-server/src/engine/core-modules/invitation/invitation.auto-resolver-opts.ts

```diff
@@ -0,0 +1,39 @@
+import {
```

Comment:
Not sure if this file. Is it really necessary?

Reply:
It can also be set in module (e.g. ObjectMetadataModule)

It's a config for nestjs-query to define which resolver should or shouldn't automatically be created so yes it's useful to configure it. 
I'm not a fan of nestjs-query because as soon as you need to add a bit of business logic then it becomes useless (you can see examples in the code, it's possible I think to inject just a service or overwrite the whole resolver). 
Depending on how you move forward, some of those resolvers can still be helpful for you (e.g. if you don't go through the workspace entity then you might want to try removing ` many: { disabled: true }`)